he.c: fix bad udelay usage

udelay() is valid for small delays. For millisecs delays, mdelay and the likes
should be used.

Signed-off-by: Arnaud Patard <arnaud.patard@rtp-net.org>
Index: mga/drivers/atm/he.c
===================================================================
--- mga.orig/drivers/atm/he.c	2013-02-19 00:58:34.000000000 +0100
+++ mga/drivers/atm/he.c	2013-04-08 08:55:55.598673595 +0200
@@ -1056,7 +1056,7 @@ static int he_start(struct atm_dev *dev)
 	he_writel(he_dev, 0x0, RESET_CNTL);
 	he_writel(he_dev, 0xff, RESET_CNTL);
 
-	udelay(16*1000);	/* 16 ms */
+	mdelay(16);	/* 16 ms */
 	status = he_readl(he_dev, RESET_CNTL);
 	if ((status & BOARD_RST_STATUS) == 0) {
 		hprintk("reset failed\n");
